pyzabbix/sender.py

```diff
@@ -385,3 +449,57 @@ def send(self, metrics):
         for m in range(0, len(metrics), self.chunk_size):
             result.parse(self._chunk_send(metrics[m:m + self.chunk_size]))
         return result
+
+    def _create_active_checks_request(self, host):
+        """Create a formatted request to zabbix from a host name.
+
+        :type host: str
+        :param host: The host to ask about
+
+        :rtype: str
+        :return: Formatted zabbix request
+        """
+        request = '{"request":"active checks","host":"%s"}' % host
+        request = request.encode("utf-8")
+        logger.debug('Request: %s', request)
+        return request
+
+    def send_active_checks(self, host):
```

Comment:
This function returns list of active checks from zabbix server, then why it calls `send_active_checks` instead of, an example, `get_active_checks`?